assets/app/scripts/misc/labelSelector.js

```diff
@@ -1,5 +1,11 @@
-function LabelSelector(selector) {
+// selector (optional) - the JSON format as returned by k8s API
+// emptySelects (optional) - whether a label selector with no conjuncts
+//      selects objects.  Typical behavior is false.  Example of an
+//      exceptional case is when filtering by labels, no label selectors
+//      means no filters.
+function LabelSelector(selector, emptySelects) {
```

Comment:
name `emptySelectsAll`? `emptySelects` sounds like it is a list of the things `empty` selected